### PR TITLE
Users/demarcey/command bar on render aria

### PR DESCRIPTION
**Pull request by dmarcey**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by JasonGore** [comment]
Change file type should be `minor` to reflect API change

**Comment by micahgodbolt** [comment]
you'll want to take another shot at this now that 6.0 has landed. The new commandBar file structure is totally different.

File: packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts [hunk 1]
```diff
@@ -451,10 +451,14 @@ export interface IContextualMenuItem {
...
-   *  This can be used to make sure that a custom menu item click dismisses the menu.
+   * This can be used to make sure that a custom menu item click dismisses the menu.
+   *
+   * posInSet and setSize are passed to the function for cases where the rendered item needs to be
+   * able to set these aria attributes (like in the command bar)
+   *
    * @defaultvalue undefined
    */
-  onRender?: (item: any, dismissMenu: (ev?: any, dismissAll?: boolean) => void) => React.ReactNode;
+  onRender?: (item: any, dismissMenu: (ev?: any, dismissAll?: boolean) => void, posInSet?: number, setSize?: number) => React.ReactNode;
```
Comment (joschect) [inline packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts]: Be careful with how this interacts with customlistlength. If someone has a list item which actually renders multiple items, the posInSet might not be accurate.

**Comment by dzearing** [comment]
Any update here?

File: packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts (hunk 1 above)
Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts]: @joschect are you asking for a change here? Or documentation update? How can we help avoid this problem you're describing?

**Comment by micahgodbolt** [comment]
btw, this code still needs to get ported over to commandBar.Base as the file structure has changed.

File: packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts (hunk 1 above)
Comment (JasonGore) [reply packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts]: There are two number arguments which are optional, which reduces arg safety quite a bit. The calling function has no idea if a caller provided only posInSet and not setSize or omitted posInSet and is specifying setSize.

Comment (JasonGore) [reply packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts]: (In addition to Jon's concerns)

**Comment by dzearing** [comment]
Closing as this has idled since May 25th. Please feel to reopen if needed.

**Pull request closed without merging**